firmware/ec/platform/oc-sdr/schema/sys_schema.json

```diff
@@ -0,0 +1,895 @@
+[
```

Comment:
@JoshuaJeyaraj please check that this file includes equivalent changes for PR 164.